test/kpod_run.bats

```diff
@@ -56,3 +56,23 @@ ALPINE="docker.io/library/alpine:latest"
     [ "$status" -eq 0 ]
 
 }
+
+@test "run environment test" {
+
+#    run bash -c "${KPOD_BINARY} ${KPOD_OPTIONS} run ${ALPINE} sh -c printenv | grep HOSTNAME"
```

Comment:
Curious why this is commented out?

Reply:
We don't set the hostname in our containers since we are not using networks yet.  As soon as we have network support for kpod run, we can uncomment this.  I will add a comment on this.